lib/tasks/after_tasks.rake

```diff
@@ -362,6 +362,26 @@ namespace :After do
 
   #### Add your new tasks here
 
+  desc "Backfill renamed_at for existing users"
```

Comment:
Can you move the new task to the end of the list?

https://github.com/otwcode/otwarchive/blob/086e6f08bd1578943cf1ddf2ca14c0cdcdb4e210/lib/tasks/after_tasks.rake#L877